saleor/discount/test_discounts.py

```diff
@@ -64,8 +64,9 @@ def test_percentage_discounts(product_variant):
 
 @pytest.mark.parametrize(
     'total, discount_value, discount_type, limit, expected_value', [
-    ('100', 10, Voucher.DISCOUNT_VALUE_FIXED, None, 10),
-    ('100.05', 10, Voucher.DISCOUNT_VALUE_PERCENTAGE, 100, 10)])
+        ('100', 10, Voucher.DISCOUNT_VALUE_FIXED, None, 10),
+        ('100.05', 10, Voucher.DISCOUNT_VALUE_PERCENTAGE, 100, 10)
+    ])
```

Comment:
The rewrap you did here is by all means correct, I'd just leave the closing parenthesis at the end of line.